### PR TITLE
Home_work_form

### PR DESCRIPTION
**Pull request by Kuptsov-Dmitriy**

### CODE REVIEW

File: dkuptsov/lesson_02/Pithon_part1.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+		<p>Введите эти 4 строки:</p>
+		<br>
+<pre>#!/usr/bin/env python
+print 'Hello. I am a python program.'
+name = raw_input("What is your name?")
+print "Hello there, " + name + "!"</pre>
+		<p>Это всё. Сохраните файл как <strong>hello.py</strong> в любом месте. Я предлагаю сохранить его в папке
+		<strong>python_examples</strong> в домашнем каталоге. Этот пример показывает, как просто написать программу на
+		Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале</p>
+          <p><strong><u><em>chmod +x hello.py</em></u></strong></p>
```
Comment (pluhin) [inline dkuptsov/lesson_02/Pithon_part1.html]: Do not use tag p for non-human content

File: dkuptsov/lesson_02/Site_Maket/Site_Maket.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        <h2>Меню список</h2>
+             <ul>
+                 <li><a href="./site_maket.html">Главная</a></li>
+                 <li><a href="./catalog.html">Каталог</a></li>
+                 <li><a href="./promo.html">Акции</a></li>
+                 <li><a href="./about.html">О нас</a></li>
+             </ul>
+        <h2>Меню с вложенным спском</h2>
+             <ul>
+                <li><a href="./site_maket.html">Главная</a></li>
```
Comment (pluhin) [inline dkuptsov/lesson_02/Site_Maket/Site_Maket.html]: Wrong structure for nested lists

**Review by pluhin — CHANGES_REQUESTED** [review]

File: dkuptsov/lesson_02/Site_Maket/Site_Maket.html (hunk 2 above)
Comment (Kuptsov-Dmitriy) [reply dkuptsov/lesson_02/Site_Maket/Site_Maket.html]: corrected.

**Comment by pluhin** [comment]
You have  conflicts in files

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**